apps/prairielearn/src/server.ts

```diff
@@ -2428,14 +2427,14 @@ if (esMain(import.meta) && config.startServer) {
           ssl: config.postgresqlSsl,
           errorOnUnusedParameters: config.devMode,
         };
-        function idleErrorHandler(err) {
+        function idleErrorHandler(err: Error, _client: sqldb.PoolClient) {
```

Comment:
If this is unused, no need to include it.

```suggestion
        function idleErrorHandler(err: Error) {
```